pkg/osd_cluster_upgrader/upgrader.go

```diff
@@ -567,3 +580,35 @@ func hasUpgradeCommenced(c client.Client, uc *upgradev1alpha1.UpgradeConfig) (bo
 
 	return true, nil
 }
+
+// nodeDrained checks if the nodes are being drained successfully during the upgrade
+func nodeDrained(c client.Client, uc *upgradev1alpha1.UpgradeConfig, logger logr.Logger, metrics metrics.Metrics) (bool, error) {
+	nodeList := &corev1.NodeList{}
+	errNode := c.List(context.TODO(), nodeList)
+	if errNode != nil {
+		return false, errNode
+	}
+
+	pdbAlertName := "PodDisruptioinBudgetAtTime"
```

Comment:
`PodDisruptionBudgetAtTime`

Reply:
@bmeng - Did you intend to refer to [PodDisruptionBudgetAtLimit](https://github.com/openshift/cluster-kube-controller-manager-operator/blob/master/manifests/0000_90_kube-controller-manager-operator_05_alert-pdb.yaml#L12) alert here instead of `PodDisruptionBudgetAtTime` ?

Reply:
thanks for catching the stupid typo...